app/routes/sitemap.js

```diff
@@ -0,0 +1,23 @@
+import express from 'express'
+import fs from 'fs'
+import path from 'path'
+import getSitemap from '../tools/getSitemap'
+
+const router = express.Router()
+
+const sitemapPath = path.resolve(__dirname, '../../sitemap.xml')
+console.log('Sitemap path:', sitemapPath)
+router.get('/sitemap.xml', (req, res) => {
+  fs.readFile(sitemapPath, (err, data) => {
```

Comment:
There should not be a sitemap.xml file.  The sitemap should be generated right here, on demand.  Much of the code in getsitemap.js should be right here.  getsitemap.js in tools is not needed.